projects/plugins/boost/app/assets/src/js/features/super-cache-info/super-cache-info.tsx

```diff
@@ -0,0 +1,164 @@
+import React, { useState } from 'react';
+import { DataSyncProvider, useDataSync } from '@automattic/jetpack-react-data-sync-client';
+import { __, sprintf } from '@wordpress/i18n';
+import { Notice, Button } from '@automattic/jetpack-components';
+import {
+	measureSuperCacheSaving,
+	isSuperCachePluginActive,
+	isSuperCacheEnabled,
+} from './lib/utils/measure';
+import { z } from 'zod';
+
+const SuperCacheInfo = () => {
+	const [ state, setState ] = useState( { status: 'idle' } ); // idle, testing, error, complete
+
+	const [ { data }, { mutate } ] = useDataSync(
+		'jetpack_boost_ds',
+		'super_cache_notice_disabled',
+		z.boolean()
+	);
+
+	const isNoticeDismissed = data;
+	const setNoticeDismissed = mutate;
+
+	const runTest = async () => {
+		setState( { status: 'testing' } );
+		try {
+			const saving = await measureSuperCacheSaving();
+			setState( { status: 'complete', saving } );
+		} catch ( error ) {
+			setState( { status: 'error', error } );
+		}
+	};
+
+	const navToSuperCacheSettings = () => {
+		window.location.href = './options-general.php?page=wpsupercache';
+	};
+
+	if ( isNoticeDismissed ) {
+		return null;
+	}
+
+	if ( ! isSuperCachePluginActive() ) {
+		return null;
+	}
+
+	if ( ! isSuperCacheEnabled() ) {
+		return (
+			<Notice
+				level="warning"
+				title={ __( 'Super Cache is installed but not enabled', 'jetpack-boost' ) }
+				actions={ [
+					<Button key="start" isPrimary onClick={ () => navToSuperCacheSettings() }>
```

Comment:
Not a big deal, but the extra `() => { ...() }` around the function name isn't necessary here as passing the raw function in fine without defining a closure around it. 

```suggestion
					<Button key="start" isPrimary onClick={ navToSuperCacheSettings }>
```

Note: the `onClose` below on line 57 _does_ need a closure around it, as you're passing a param to the function when the closure gets called.

Reply:
Thanks!